src/main/java/org/jitsi/videobridge/cc/BitrateController.java

```diff
@@ -1195,6 +1187,11 @@ public void addPayloadType(PayloadType payloadType)
     {
         payloadTypes.put(payloadType.getPt(), payloadType);
         adaptiveTrackProjections.forEach(atp -> atp.addPayloadType(payloadType));
+
+        if (payloadType.getEncoding() == PayloadTypeEncoding.RTX)
```

Comment:
Do we not signal an RTX payload type with firefox?

Reply:
no